src/sasctl/pzmm/write_json_files.py

```diff
@@ -51,7 +75,12 @@ def writeVarJSON(cls, inputData, isInput=True, jPath=Path.cwd()):
                     else:
                         isStr = False
 
-                if isStr:
+            if isStr:
+                outputLevel = "nominal"
+                outputType = "string"
+                outputLength = predict.str.len().max()
```

Comment:
outputLevel, outputType, and outputLength appear to never actually be used.  Also, what is `predict` - I do not see it defined or passed as a parameter.

Reply:
Something changed here that should not have. I'm going to parse through this section and see what is going on and where things broke down. Likely just revert it back to a prior state.